doc: describe specifying workload attributes with service levels

annastuchlik opened the pull request:
Fix https://github.com/scylladb/scylladb/issues/11197

This PR adds a new page where specifying workload attributes with service levels is described and adds it to the menu.

Also, I had to fix some links because of the warnings.

psarna commented on docs/glossary.rst:
```diff
@@ -176,3 +176,7 @@ Glossary
        Cache dummy rows are entries in the row set, which have a clustering position, although they do not represent CQL rows written by users.  Scylla cache uses them to mark boundaries of population ranges, to represent the information that the whole range is complete, and there is no need to go to sstables to read the gaps between existing row entries when scanning.
+      
+    Workload
+      A database category that allows you to manage different sources of database activities, such as requests or administrative activities. By creating workloads, you can specify how ScyllaDB will process those activities. For example, you can prioritize one workload over another (e.g., user requests over administrative activities). See :doc:`Workload Prioritization </using-scylla/workload-prior ...
```
1. nitpick: I think that "By creating workloads" should be replaced with "By defining workloads", it fits the context a little better
2. Prioritizing one workload over another is an enterprise-only feature, and I don't know if in these open-source docs we try to avoid mentioning enterprise-only features - do we?

tzach commented on docs/upgrade/_common/upgrade-guide-from-5.0-to-2022.1-ubuntu-and-debian.rst:
```diff
@@ -30,7 +30,7 @@ Apply the following procedure **serially** on each node. Do not move to the next
 
 * Not to use new 2022.1 features.
-* Not to run administration functions, like repairs, refresh, rebuild or add or remove nodes. See :doc:`here </operating-scylla/manager/2.1/sctool>` for suspending Scylla Manager's scheduled or running repairs.
+* Not to run administration functions, like repairs, refresh, rebuild or add or remove nodes. See `sctool <https://manager.docs.scylladb.com/stable/sctool/>`_ for suspending Scylla Manager's scheduled or running repairs.
```
Look like a mix in commits
Not related to the PR

annastuchlik replied:
@tzach I had to update it to avoid conflicts and another complicated rebasing of my brach.

annastuchlik commented on docs/using-scylla/workload-attributes.rst:
```diff
@@ -0,0 +1,132 @@
+=============================
+Defining Workload Attributes
```
@tzach In this article, I focused on workloads because the presentations during the summit implied that this is why the feature was added. 

"Service Level Attributes Properties" is a four-noun phrase, which makes it unclear. Also, it doesn't indicate what the user may want to achieve (our documentation should be more goal-oriented).
To make the title less workload-related, we could change it to "**Defining Service Level Attributes**".
What do you think?

eliransin reviewed (changes requested):
Sorry for the late review.
This is by far the clearest explanation of workload prioritization I have ever seen.
Just one more comment which I am not sure if relevant:
The service level mechanism in OSS doesn't include workload prioritization which is an attribute of a service level which allows you to prioritize and isolate two workloads. So I am not sure if it should be referenced from here or from the enterprise documentation if there is such a thing.

tzach commented:
LGTM

annastuchlik commented:
@eliransin Thanks for reviewing! I've updated the syntax - could you re-review it?

About linking to the Enterprise docs: now there is one documentation set for ScyllaDB Open Source and Enterpise, so we can cross-reference different sections. Once we split the docs, numerous structure changes will be required, but it's not the scope of this PR. I'll create an issue to update the service level-related content.

eliransin reviewed (approved):
LGTM.
10x

annastuchlik commented:
@scylladb/scylla-maint Please backport to 5.1
